tools/slurm/train.slurm.sh

```
@@ -0,0 +1,47 @@
+echo "===== SIMBA Training: th20_asb ====="
+echo "Job ID: $SLURM_JOB_ID"
+echo "Node:   $SLURM_NODELIST"
+echo "Date:   $(date)"
+nvidia-smi
+
+PREPRO_DIR=/mnt/data2/nkubrakov/massspecgym/preprocessing_th20_asb
+CHECKPOINT_DIR=/mnt/data2/nkubrakov/massspecgym/checkpoints_th20_asb
+
+mkdir -p "$CHECKPOINT_DIR"
+
+cd /home/nkubrakov/simba
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)